app/mailers/welcome_mailer.rb

```diff
@@ -1,7 +1,7 @@
 class WelcomeMailer < ApplicationMailer
   def welcome_notification(user)
     mail(
-      to: email_with_name(user.email_address, user.full_name),
+      to: email_to_user(user),
```

Comment:
I've been wondering if there are potential benefits of sending emails 'to' the formatted email / name.